libs/datamodel/connectors/dml/src/native_type_instance.rs

```diff
@@ -1,19 +1,25 @@
-use native_types::NativeType;
+use native_types::{MsSqlType, NativeType, TypeParameter};
 use serde::de::DeserializeOwned;
+use std::convert::TryFrom;
 
 /// represents an instance of a native type declared in the Prisma schema
 #[derive(Debug, Clone, PartialEq)]
 pub struct NativeTypeInstance {
     /// the name of the native type used in the Prisma schema
     pub name: String,
     /// the arguments that were provided
-    pub args: Vec<u32>,
+    pub args: Vec<TypeParameter>,
```

Comment:
`TypeParameter` is a data structure specific to SQL Server and we shouldn't use it in this generic place. I see 2 options:
1. As far as I can tell from the code we should be fine with a `Vec<String>`.
2. How about we introduce a new enum `NativeTypeParameter` with the variants `::Number` and `::Literal`?

Reply:
2.